source/fab/builder.py

```diff
@@ -27,6 +26,72 @@
 from fab.queue import QueueManager
 
 
+def entry() -> None:
+    """
+    Entry point for the Fab build tool.
+    """
+    import argparse
+    import multiprocessing
+    import sys
+    import fab
+
```

Comment:
This looks strange to me - I'm surprised `flake8` is okay with it?  Certainly `pylint` will throw an absolute wobbly if you have `import`s that aren't at the top level.  What is the advantage of them being here?

Reply:
Importing is bound to scope like everything else. That means these libraries are only visible in the subroutine they are imported to. I chose to do this as they aren't needed anywhere else. It seemed neater.